theano/gof/graph.py

```diff
@@ -125,8 +125,12 @@ def run_params(self):
         Returns the params for the node, or NoParams if no params is set.
 
         """
-        if hasattr(self.op, 'get_params'):
-            return self.op.get_params(self)
+        if hasattr(self.op, 'params_type'):
```

Comment:
We never supported defining only get_params.  If you had get_params defined, the rest of the code assumed that you also had params_type and this was documented.  Please don't introduce support for that.